app/src/test/java/com/hapramp/CommunitiesUnitTest.java

```diff
@@ -57,4 +58,11 @@ public void testCleanContent() {
     System.out.println(com.hapramp.utils.StringUtils.stringify(jsonMetadata.getJson()));
   }
 
+  @Test
```

Comment:
This test makes no sense. Please **test** the permlink generated here and don't simply print it.